packages/core/integration-tests/test/css.js

```diff
@@ -1,91 +1,65 @@
 const assert = require('assert');
 const path = require('path');
 const fs = require('@parcel/fs');
-const {bundle, run, assertBundleTree, rimraf, ncp} = require('./utils');
-
-describe.skip('css', function() {
-  it('should produce two bundles when importing a CSS file', async function() {
+const {
+  bundle,
+  run,
+  assertBundles,
+  assertBundleTree,
+  rimraf,
+  ncp
+} = require('./utils');
+
+describe('css in v2', () => {
+  it('should produce two bundles when importing a CSS file', async () => {
     let b = await bundle(path.join(__dirname, '/integration/css/index.js'));
 
-    await assertBundleTree(b, {
-      name: 'index.js',
-      assets: ['index.js', 'index.css', 'local.js', 'local.css'],
-      childBundles: [
-        {
-          name: 'index.map'
-        },
-        {
-          name: 'index.css',
-          assets: ['index.css', 'local.css'],
-          childBundles: [
-            {
-              type: 'map'
-            }
-          ]
-        }
-      ]
-    });
+    await assertBundles(b, [
+      {
+        name: 'index.js',
+        assets: ['index.js', 'local.js']
+      },
+      {
+        name: 'local.css',
+        assets: ['index.css', 'local.css']
+      }
+    ]);
 
     let output = await run(b);
     assert.equal(typeof output, 'function');
     assert.equal(output(), 3);
   });
 
-  it('should support loading a CSS bundle along side dynamic imports', async function() {
+  // Currently failing as Windows produces different hashes
```

Comment:
should we try to address this and achieve consistent hashes across build platforms? Tools might behave differently and make this unpredictable so I'm not sure it's realistic.

Probably for these tests we should disable content digests or teach `assertBundle` to ignore them?